.github/workflows/benchmarking.yml

```diff
@@ -417,7 +417,20 @@ jobs:
         test_selection: performance
         run_in_parallel: false
         save_perf_report: ${{ env.SAVE_PERF_REPORT }}
-        extra_params: -m remote_cluster --timeout 21600 -k test_pgbench_remote_pgvector
+        extra_params: -m remote_cluster --timeout 21600 -k test_pgbench_remote_pgvector_hnsw
+      env:
+        BENCHMARK_CONNSTR: ${{ steps.set-up-connstr.outputs.connstr }}
+        VIP_VAP_ACCESS_TOKEN: "${{ secrets.VIP_VAP_ACCESS_TOKEN }}"
+        PERF_TEST_RESULT_CONNSTR: "${{ secrets.PERF_TEST_RESULT_CONNSTR }}"
+
+    - name: Benchmark pgvector halfvec queries
```

Comment:
We don't need to create a separate step for this and can set `test_selection: performance/test_perf_pgvector_queries.py` in the previous step (need to remove `-k test_pgvector_indexing` filter then). But it's completely optional and up to you